test/shared/src/main/scala/zio/test/mock/Method.scala

```diff
@@ -24,15 +24,15 @@ import zio.test.Assertion
  * A `Model[M, I, A]` represents a capability of module `M` that takes an
  * input `I` and returns an effect that may produce a single `A`.
  */
-trait Method[M, I, A] { self =>
+trait Method[-M, I, +A] { self =>
 
   /**
    * Provides the `Assertion` on method arguments `I` to produce `ArgumentExpectation`.
    *
    * Available only for methods that do take arguments.
    */
   @silent("parameter value ev in method apply is never used")
-  def apply(assertion: Assertion[I])(implicit ev: I =!= Unit): ArgumentExpectation[M, I, A] =
+  def apply[A1 >: A](assertion: Assertion[I])(implicit ev: I =!= Unit): ArgumentExpectation[M, I, A1] =
```

Comment:
I don't think the type parameter is necessary here since `A` doesn't appear anywhere in the method signature.

Reply:
You are right.